<launch>

  <node name="publisher_node" pkg="face_classification" type="video_emotion_color_demo_qt.py" output="screen">
    <param name="detection_model_path" value="$(find face_classification)/trained_models/detection_models/haarcascade_frontalface_default.xml"/>
    <param name="emotion_model_path" value="$(find face_classification)/trained_models/emotion_models/fer2013_mini_XCEPTION.110-0.65.hdf5"/>
  </node>
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>
  <node name="qt_tts" pkg="qt_tts" type="qt_tts_festivalwrapper.py" output="screen" />
  <node name="qt_face" pkg="qt_face" type="face_duration.py" output="screen" />

</launch>
